### ISSUE TITLE
Landing page is missing

### ISSUE THREAD

**Issue by ghost** [openzim/zim-requests#1441]
Empty page with only a footer legal disclaimer

<s>https://library.kiwix.org/wiktionary_ay_all_nopic</s>
https://library.kiwix.org/wikiquote_ga_all_nopic
https://library.kiwix.org/wikivoyage_es_all_maxi
https://library.kiwix.org/wikivoyage_ps_all_maxi
https://library.kiwix.org/wikivoyage_es_all_nopic
https://library.kiwix.org/wikivoyage_es_all_maxi
<s>https://library.kiwix.org/wikibooks_ky_all_nopic
https://library.kiwix.org/wiktionary_ky_all_nopic
https://library.kiwix.org/wiktionary_ks_all_nopic
https://library.kiwix.org/wikipedia_diq_all_nopic</s>
https://library.kiwix.org/wikivoyage_ps_all_nopic

**Comment by benoit74** [comment]
What is left to be fixed, looks like the recent 1.14/1.15 runs fixed them all ? Everything is fine from my PoV, shall I close the issue?

**Comment by Popolechien** [comment]
Tested a couple, LGTM

**Comment by kelson42** [comment]
> What is left to be fixed, looks like the recent 1.14/1.15 runs fixed them all ? Everything is fine from my PoV, shall I close the issue?

I don't think so, https://browse.library.kiwix.org/content/wikiquote_ga_all_nopic/A/Pr%C3%ADomh%C2%ADleathanach is empty for example!

**Comment by benoit74** [comment]
Just like online page: https://ga.wikiquote.org
